app/components/roadmap-page/-utils/features.yaml

```diff
@@ -273,8 +273,8 @@ features:
       Once the extension point exists and we can experiment with different
       approaches, the next step will be to evaluate when and how this might
       become part of the default out-of-the-box Ember experience.
-    status: under-consideration
-    statusText: 'RFC Submitted'
+    status: in-development
```

Comment:
I feel like some changes to the description should accompany status changes, so people can track where the implementation is happening.

Reply:
@twokul you can help us by updating the field description? please